README.md

```diff
@@ -1,2 +1,20 @@
 # microverse-hello-world-project
-My first project for Hello World day 2 activity.
+Hello Microverse project that contains a HTML with an header and a css file that colors the header.
+This project is part of an activity that helps me to get aquainted with the GitHub flow, Linters and pull requests. The project was made using the HTML and CSS languages. These technologies were used because the project was a webpage with a title. I then had to add a .gitignore file that ignored the files I did not want to upload. A challenge faced was the styling that was required in the CSS file.
+
+Built With
+HTML & CSS
+
+Getting Started
+There are no installations required. Simply download the files and open the index.html to open the webpage.
+
+Authors
+GitHub: @the-catalyst
+
+Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
```

Comment:
- [ ] As required by the previous reviewer, your readme should be descriptive and ordered. Kindly use [this](https://github.com/microverseinc/readme-template) template for your readme, customising it to suit your needs.

Reply:
Is it the readme overall or the specified line? I have followed the template and customized it as necessary, am I expected to include all the unnecessary headings in the template? Or should I have more information on the specified line?